uint16[] id_list
float32[] input_pos_list
uint16[] interpolation_freq_list
